runtime/tests/interpreter/metatype_test.go

```diff
@@ -82,3 +82,84 @@ func TestInterpretMetaType(t *testing.T) {
 		)
 	})
 }
+
+func TestInterpretIsInstance(t *testing.T) {
+
+	t.Parallel()
+
+	cases := []struct {
```

Comment:
I added more test cases and restructured the test cases a bit so that it's easy to add and review them

Reply:
@zhangchiqing Thanks for the additional test cases. 

To make it easier to debug the tests, can you please give them names and run them as sub-tests using `t.Run`. See e.g. https://dave.cheney.net/2019/05/07/prefer-table-driven-tests "Introducing sub tests"